.circleci/config.yml

```diff
@@ -154,7 +154,7 @@ microsite: &microsite
             nvm install
             nvm use
             node -v
-            ./sbt coreJVM/doc coreJS/doc streamsJVM/doc streamsJS/doc
+            ./sbt coreJVM/doc coreJS/doc streamsJVM/doc streamsJS/doc testJVM/doc testJS/doc
```

Comment:
You can remove this whole line, it’s actually useless here.

Reply:
Okay.